### ISSUE TITLE
Questions about disparity symbols and loss functions

### ISSUE THREAD

**Issue by diligentwxx** [TomTomTommi/bidastereo#7]
I found that the final disparity output by the bidastereo model is **negative** instead of positive, which may be due to the fact that
` coords = self.coords + flow` when TFCL was built, so absolute values were used in forward_batch_test:
`predictions["disparity"] = (torch.cat(disp_preds).squeeze(1).abs())[:, :1]`.
However, during training, the loss is calculated in this way :`i_loss = (flow_pred - flow_gt).abs()` , and the absolute value is not used. Since the `flow_gt` is positive and `disp_preds` is negative, will this affect the convergence of the model itself ?  Or is my understanding wrong? I look forward to hearing from you. Thanks.

**Comment by diligentwxx** [comment]
I found that the `flow_gt` was also negative, and used absolute values for all evaluations. I got it wrong, thank you.
